husky_components/auto-complete/main.js

```diff
@@ -71,10 +71,13 @@ define([], function() {
     templates = {
         main: [
             '<div class="husky-auto-complete <%= dropdownSizeClass %>">',
-                '<div class="front">',
-                    '<a class="fa-<%= autoCompleteIcon %>"></a>',
-                '</div>',
-                '<div class="input"></div>',
+            '    <div class="front">',
+            '       <a class="fa-<%= autoCompleteIcon %>"></a>',
+            '    </div>',
+            '    <div class="input"></div>',
+            '    <div class="back">',
```

Comment:
Forget it for now, the other class is also called front, and it was probably like that previously.